src/actions/config.js

```diff
@@ -7,6 +7,12 @@ export const CONFIG_REQUEST = "CONFIG_REQUEST";
 export const CONFIG_SUCCESS = "CONFIG_SUCCESS";
 export const CONFIG_FAILURE = "CONFIG_FAILURE";
 
+const configUrl =
+  get(
+    get(document.querySelectorAll('head link[rel="cms-config-url"][type="text/yaml"], [type="application/x-yaml"]')[0],
```

Comment:
I don't believe `head link[rel="cms-config-url"][type="text/yaml"], [type="application/x-yaml"]` is valid, since you are checking for either `head link[rel="cms-config-url"][type="text/yaml"]` or `[type="application/x-yaml"]`.

Reply:
@erquhart To be honest, I think we should check the format separately from the selector, so that it is easier to add support for multiple formats later.

Reply:
@brianlmacdonald

Reply:
Two thoughts:

1. Just use `querySelector` and take the first one. I don't think we need robust handling here - it's an advanced use case, they should just make sure they don't add more than one.
2. It's cool to check the `type` property of the retrieved element directly, a bit cleaner too.